packages/create-remix/templates/vercel/remix.config.js

```diff
@@ -2,9 +2,11 @@
  * @type {import('@remix-run/dev/config').AppConfig}
  */
 module.exports = {
-  appDirectory: "app",
-  assetsBuildDirectory: "public/build",
-  publicPath: "/build/",
-  serverBuildDirectory: "api/_build",
+  serverBuildTarget: "vercel",
```

Comment:
Let's use node-cjs here in dev instead of branching in the `remix dev` command code